Sources/Plasma/Apps/plClient/Mac-Cocoa/main.mm

```diff
@@ -457,20 +449,27 @@ - (void)startClient
 {
     PF_CONSOLE_INITIALIZE(Audio)
 
-    self.plsView.delegate = self;
+    self.renderLayer = gClient->GetPipeline()->GetRenderLayer();
+    _renderLayer.backgroundColor = NSColor.blackColor.CGColor;
+
+    self.window.contentView.layer = _renderLayer;
+    self.window.contentView.wantsLayer = YES;
```

Comment:
Not sure how I feel. PLSView was meant to be self contained and wrap all details about the layer. Mostly.

Reply:
There's some nuance here that I don't entirely understand about layer-hosted views and layer-backed views and the order of setting `layer` and `wantsLayer`